Python/graphs/Knight_Walk.py

```diff
@@ -0,0 +1,100 @@
+# Importing sys module to initalize the maxsize
```

Comment:
@atriadhiakri2000 In this code lot of whitespaces is there please remove the whitespaces.